configs/config_uecc.h

```diff
@@ -0,0 +1,3433 @@
+/**
+ * \file config.h
```

Comment:
I don't think we need this file here (we already have `configs/baremetal.h` in the base branch, which this branch should adjust to use the micro-ecc option (and remove things specific to our own ECC implementation), but even if we did, it should not be a full copy of the main config.h (see how the other files in this directory look).

Reply:
True, this was done before the baremetal branch existed.